packages/uikit/src/components/SubMenuItems/SubMenuItems.tsx

```diff
@@ -98,6 +99,11 @@ const SubMenuItems: React.FC<React.PropsWithChildren<SubMenuItemsProps>> = ({
                   >
                     {Icon && <Icon color={isActive ? "secondary" : "textSubtle"} mr="4px" />}
                     {label}
+                    {label === "Buy Crypto" ? (
+                      <Button height="25px" px="8px" ml="10px">
+                        New
+                      </Button>
+                    ) : null}
```

Comment:
Try using the same logic as menu config.
Avoid harcode

Reply:
resolved